[CON-26] feat: Add Airtable guide (#109)
amp-labs/docs: src/provider-guides/airtable.mdx

- **Proxy Actions**, using the base URL `https://api.airtable.com`.

## Before You Get Started

To integrate Airtable with Ampersand, you will need an [Airtable](https://airtable.com) Account.

Once your account is created, you'll need to register an OAuth app and obtain the following credentials:

- Client ID
- Client Secret
- Scopes

You will then use these credentials to connect your application to Ampersand.

### Create an Airtable Account

Here's how you can sign up for a Airtable account:

- Go to the [Airtable Site](https://airtable.com/signup) and sign up for a free account.
- Sign up using your preferred method and complete any necessary verification.

### Creating an Airtable OAuth App

Follow the steps below to create an Airtable OAuth app:

1. Login to [Airtable](https://airtable.com/login) using your credentials.
2. Go to the [Airtable's Builders Hub](https://airtable.com/create/oauth).
3. Click on **Register New Oauth Integration**.
4. Enter the following details:

   1. **Name:** Choose a name for your application.
   2. **OAuth Redirect URL:** Enter `https://api.withampersand.com/callbacks/v1/oauth`.
   3. Click on **Register Integration**.

> 5. You will see the **Client ID**, Click on **Generate client secret** to generate client secret. Note these credentials, as you will need them to connect your app to Ampersand.

Reviewer: ```suggestion
5. You will see the **Client ID** here. Click on **Generate client secret** to generate the client secret. Keep these credentials handy to connect your app to Ampersand later.
```

> 6. Select the appropriate permissions(scopes) for your application.

Reviewer: ```suggestion
6. Select the appropriate scopes for your application.
```

> 7. Provide a support email, privacy policy, and terms of service.
> 8. Click **save changes**.

Reviewer: ```suggestion
7. Complete the rest of the setup & click on **Save changes**.
```
